src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -95,14 +93,16 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, MSBuild.TaskLog
             return !Log.HasLoggedErrors;
         }
 
-        public async Task UploadAssets(ITaskItem item, SemaphoreSlim clientThrottle, bool allowOverwrite = false)
+        public async Task UploadAssets(ITaskItem item, SemaphoreSlim clientThrottle, bool allowOverwrite = false, bool symbols = false)
         {
             string relativeBlobPath = item.GetMetadata("RelativeBlobPath");
-            if (string.IsNullOrEmpty(relativeBlobPath))
+
+            if (string.IsNullOrEmpty(relativeBlobPath) || symbols)
             {
+                string symbolsFolder = symbols ? "symbols/" : string.Empty;
```

Comment:
No reason to special case this here. Just set the RelativeBlobPath metadata to "symbols/$(filename)" in the root directory below.